app/build.gradle

```diff
@@ -218,6 +219,8 @@ dependencies {
     implementation 'androidx.viewpager2:viewpager2:1.1.0-beta01'
     implementation "androidx.work:work-runtime-ktx:${androidxWorkVersion}"
     implementation "androidx.work:work-rxjava3:${androidxWorkVersion}"
+    implementation "androidx.paging:paging-runtime:${androidxPagingVersion}"
```

Comment:
Please keep the AndroidX libraries in alphabetical order (applies to the ext block too).